betterlockscreen

```diff
@@ -120,6 +120,18 @@ lockselect() {
 	postlock
 }
 
+logical_px() {
+	# get dpi value from xrdb
+	local DPI=$(xrdb -query | awk '/Xft.dpi/ {print $2}')
+	local SCALE=$(echo "scale=2; 150 / 96.0" | bc)
```

Comment:
Don't you want to use `$DPI` here?

Reply:
Oh yes you're right, I forgot to remove this test, thank ! :+1: